Sources/ArcGISToolkit/Components/Offline/PreplannedMapModel.swift

```diff
@@ -313,6 +343,21 @@ extension FileManager {
         offlineMapAreasDirectory.appending(path: portalItemID.rawValue, directoryHint: .isDirectory)
     }
     
+    /// Calculates the size of a directory and all its contents.
+    /// - Parameter url: The directory's URL.
+    /// - Returns: The total size in bytes.
+    func sizeOfDirectory(at url: URL) -> Int {
+        guard let enumerator = enumerator(at: url, includingPropertiesForKeys: [.fileSizeKey]) else { return 0 }
+        var size = 0
```

Comment:
I would suggesting naming this `accumulatedSize` and then the other one (`fileSize`) could just be named `size`.